modules/data-catalog-tag/README.md

```diff
@@ -20,7 +22,18 @@ module "data-catalog-tag" {
       location   = "europe-west1"
       template   = "projects/project-datagov/locations/europe-west1/tagTemplates/demo"
       fields = {
-        source = "DB-1"
+        source = {
```

Comment:
Covering `dispaly_name` here would also be helpful, Can help with inventory generation for that test case, as it would be great to have it here.

Reply:
Display name is an output of the resource. Removed in the main.tf, no need to add it here.